src/navigators/RootStack.tsx

```diff
@@ -28,14 +29,17 @@ export function RootStack() {
     isLoading: isLoadingProject,
     isFetched: isFetchedProject,
   } = useActiveProject();
+  const { inviteParams } = usePendingInvite();
 
   const { useShouldRenderConsentScreen } = useConsent();
   const { shouldRenderConsentScreen, isLoading: loadingConsents } =
     useShouldRenderConsentScreen();
 
   const loadingProject = !isFetchedProject || isLoadingProject;
   const loadingAccount = !isFetchedAccount || isLoadingAccount;
-  const loadingAccountOrProject = loadingProject || loadingAccount;
+  const hasAccount = !loadingAccount && account?.id;
+  const loadingAccountOrProject =
+    loadingAccount || (hasAccount && loadingProject);
```

Comment:
`(hasAccount && loadingProject)` covers the case where the user is part of NO accounts.  In that case, we don't want to wait on `loadingProject` as it's indefinitely true due to accountHeaders never being set.

Reply:
Oh I see, this is the escape hatch I was missing. Nice.